### PR TITLE
chore: update k8s test matrix for chart updates

### PR DESCRIPTION
**Pull request by zhill**
Signed-off-by: Zach Hill <zach@anchore.com>

### CODE REVIEW

**Review by Btodhunter — DISMISSED** [review]
lgtm

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetesVersion: ["v1.14.10", "v1.22.0"]
+        kubernetesVersion: ["v1.19.0", "v1.22.0", "v1.25.0"]
```
Comment (Btodhunter) [inline .github/workflows/test.yaml:11]: I've run into issues with older versions of the `kindest:node` image, the testing for v1.19.0 might pass if you use the latest kindest image `v1.19.16`

Comment (zhill) [reply .github/workflows/test.yaml]: I'm trying 1.21 instead which is the oldest version EKS supports. You ok with 1.21, 1.22, 1.25?

Comment (zhill) [reply .github/workflows/test.yaml]: nvm, there is no 1.21.0 kindest/node image so I'm using 1.19.16 anyway.

**Comment by Btodhunter** [comment]
lgtm

**Review by Btodhunter — APPROVED** [review]

**Pull request merged**